core/modules/widgets/checkbox.js

```diff
@@ -3,129 +3,134 @@ title: $:/core/modules/widgets/checkbox.js
 
 \*/
-(function(){
+(function() {
 
 /*jslint node: true, browser: true */
 /*global $tw: false */
 "use strict";
-
 var Widget = require("$:/core/modules/widgets/widget.js").widget;
 
-var CheckboxWidget = function(parseTreeNode,options) {
-	this.initialise(parseTreeNode,options);
-};
-
 /*
 Inherit from the base widget class
 */
+var CheckboxWidget = function(parseTreeNode, options) {
+	this.initialise(parseTreeNode, options);
+};
+
 CheckboxWidget.prototype = new Widget();
 
 /*
 Render this widget into the DOM
 */
-CheckboxWidget.prototype.render = function(parent,nextSibling) {
-	// Save the parent dom node
+CheckboxWidget.prototype.render = function(parent, nextSibling) {
 	this.parentDomNode = parent;
-	// Compute our attributes
 	this.computeAttributes();
-	// Execute our logic
 	this.execute();
-	// Create our elements
 	this.labelDomNode = this.document.createElement("label");
-	this.labelDomNode.setAttribute("class",this.checkboxClass);
+	this.labelDomNode.setAttribute("class", this.checkboxClass);
 	this.inputDomNode = this.document.createElement("input");
-	this.inputDomNode.setAttribute("type","checkbox");
+	this.inputDomNode.setAttribute("type", "checkbox");
 	if(this.getValue()) {
-		this.inputDomNode.setAttribute("checked","true");
+		this.inputDomNode.setAttribute("checked", "true");
 	}
 	this.labelDomNode.appendChild(this.inputDomNode);
 	this.spanDomNode = this.document.createElement("span");
 	this.labelDomNode.appendChild(this.spanDomNode);
-	// Add a click event handler
-	$tw.utils.addEventListeners(this.inputDomNode,[
-		{name: "change", handlerObject: this, handlerMethod: "handleChangeEvent"}
-	]);
-	// Insert the label into the DOM and render any children
-	parent.insertBefore(this.labelDomNode,nextSibling);
-	this.renderChildren(this.spanDomNode,null);
+	$tw.utils.addEventListeners(this.inputDomNode, [{
+		name: "change",
+		handlerObject: this,
+		handlerMethod: "handleChangeEvent"
+	}]);
+	parent.insertBefore(this.labelDomNode, nextSibling);
+	this.renderChildren(this.spanDomNode, null);
 	this.domNodes.push(this.labelDomNode);
 };
 
+/*
+Determine if checkbox state matches the value/tag
+*/
 CheckboxWidget.prototype.getValue = function() {
-	var tiddler = this.wiki.getTiddler(this.checkboxTitle);
+	var tiddler = this.wiki.getTiddler(this.checkboxTitle),
+		value,
+		HasTag;
 	if(tiddler) {
 		if(this.checkboxTag) {
-			if(this.checkboxInvertTag) {
-				return !tiddler.hasTag(this.checkboxTag);
-			} else {
-				return tiddler.hasTag(this.checkboxTag);
-			}
-		}
-		if(this.checkboxField) {
-			var value = tiddler.fields[this.checkboxField] || this.checkboxDefault || "";
-			if(value === this.checkboxChecked) {
-				return true;
-			}
-			if(value === this.checkboxUnchecked) {
-				return false;
-			}
+			HasTag = tiddler.hasTag(this.checkboxTag);
+			return(this.checkboxInvertTag) ? !HasTag : HasTag;
 		}
+		value = (this.checkboxField) ? tiddler.fields[this.checkboxField]
```

Comment:
please make it `if else` and only use **one return** statement ...at least, I find it more clear since you're using `value` already... and perhaps call it `state` rather than `value`